index.js

```diff
@@ -56,8 +56,8 @@ const create = (win, options) => {
 				label: 'Search with Google',
 				visible: hasText,
 				click() {
-					const query = 'https://www.google.com/search?q=';
-					electron.shell.openExternal(query + props.selectionText);
+					const url = new URL('https://www.google.com/search?q=' + props.selectionText).href;
```

Comment:
This is still not correct. I asked for the query string to be properly encoded. You do that with `URL#searchParams`.